showpass/Showpass-E2E-tests
# TA-27-Profile - Verify the 'Password' form overall appearance

**Description:**
This spec will be verifying the following element presence on the ‘Password’ page.
- Page header (Password)
- Labels: “Old Password”, “New Password”, “Confirm New Password”
- Input fields: “Old Password”, “New Password”, “Confirm New Password”
- Label: “Password Strength: 0%"
- Password progress bar
- “Save” button

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-27

**How to test:**
Run the following spec and ensure all tests pass.
npx cypress run --spec "cypress/e2e/user-settings/profile.cy.js"

**Test execution results:**
![Password-appearance-passed](https://github.com/showpass/Showpass-E2E-tests/assets/138615449/b53be09f-d0e0-487b-95f2-8b4624583613)
